PlatformConfig.mk

```diff
@@ -74,15 +74,18 @@ BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX_LOCATION := 1
 
 TARGET_NO_RECOVERY := true
 
-# https://source.android.com/devices/bootloader/partitions/generic-boot#combinations, "Launch device without recovery partition":
+# https://source.android.com/devices/bootloader/partitions/generic-boot#combinations, "Launch device with A/B recovery partition":
 BOARD_USES_RECOVERY_AS_BOOT :=
 BOARD_USES_GENERIC_KERNEL_IMAGE := true
-BOARD_MOVE_RECOVERY_RESOURCES_TO_VENDOR_BOOT := true
+BOARD_MOVE_RECOVERY_RESOURCES_TO_VENDOR_BOOT :=
```

Comment:
Same for this, can probably be yeeted. You omitted `PRODUCT_BUILD_RECOVERY_IMAGE` as well, instead of `true`/empty-initialize after all.